src/components/CourtOrderPoa/package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "@bcrs-shared-components/court-order-poa",
+  "version": "0.0.0",
+  "publishConfig": {
+    "access": "public"
+  },
+  "dependencies": {
+    "@bcrs-shared-components/interfaces": "^1.0.14",
```

Comment:
Verify that this gets updated to 1.0.15 after you run the Lerna commands.

Reply:
k